infrastructure/lambda/template.yaml

```diff
@@ -220,6 +223,14 @@ Resources:
                 - ssm:GetParametersByPath
               Resource:
                 - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${AWS::StackName}/clients/*"
+        - Statement:
+            - Sid: auditEventQueueKmsEncryptionKeyPermission
+              Effect: Allow
+              Action:
+                - 'kms:Decrypt'
+                - 'kms:GenerateDataKey'
+              Resource:
+                - !ImportValue AuditEventQueueEncryptionKeyArn
```

Comment:
So this is using the `root account` permission to make this happen see https://github.com/alphagov/di-ipv-cri-address-infrastructure/blob/main/sqs/template.yaml#L76 

At the moment, that way more permission than is needed